playbooks/common/openshift-glusterfs/config.yml

```diff
@@ -10,7 +10,7 @@
         installer_phase_glusterfs: "In Progress"
       aggregate: false
 
-- name: Open firewall ports for GlusterFS nodes
+- name: Configure GlusterFS hosts
```

Comment:
I not sure if this part is right. i mean usage of `include_role` and how its being used in 3.6.

Reply:
Can you ellaborate on that? Is something failing for you in testing?

Reply:
Not a failures. Just I remember we didn't use `import_playbook` before in 3.6, but now I can see a lot of backported changes (not only gluster) using this syntax. And it requires newer ansible version for it. But as we have etcd and master playbooks using those now in 3.6, we should be fine (assumption here).